### ISSUE TITLE
pytorch Version

### ISSUE THREAD

**Issue by jiaszhang** [facebookresearch/video-long-term-feature-banks#42]
Hello, I need to compile pytorch, could you tell me the pytorch version the project ued? thank you!

**Comment by chaoyuaw** [comment]
Hi, Sorry for the late reply. I don't remember the exact version I used when I  developed this repository (April 2019), but I used the latest version back then, so my best guest would be around v1.0. Please let me know if you have any issue using this repo. Thanks!

**Comment by luuckiest** [comment]
I have problem installing the newest version of pytorch. 
Ubuntu is 18.04,
Cuda=9.0
Cudnn = 7.5
gcc = 5.5
python =2.7.x


If you could let me know which version are you using?

**Comment by chaoyuaw** [comment]
Hi, I'm sorry but I don't have access to the environment I used to develop this repo, so I don't have answers to your questions. However, if it's a PyTorch installation question (as opposed to a LFB-specific question), maybe "https://github.com/pytorch/pytorch/issues" would be helpful.

**Comment by Holmes-GU** [comment]
@chaoyuaw Hi, I am reading your codes and I am not sure about the scale operation, in the code, there is theta_phi = model.Scale(theta_phi, theta_phi, scale=latent_dim**-.5). What dose it mean? Is every element divided by scale? Thank you.

**Comment by chaoyuaw** [comment]
Yes, it's element-wise scaling by a factor of (1 / sqrt(latent_dim)). We follow Sec-3.2.1 & footnote 4 of https://arxiv.org/pdf/1706.03762.pdf for this design choice.

**Comment by Holmes-GU** [comment]
Thank you very much.


| |
SJTU 顾智浩
邮箱：gzhmsy@126.com
|

签名由 网易邮箱大师 定制

On 07/15/2020 12:25, Chao-Yuan Wu wrote:

Yes, it's element-wise scaling by a factor of (1 / sqrt(latent_dim)). We follow Sec-3.2.1 & footnote 4 of https://arxiv.org/pdf/1706.03762.pdf for this design choice.
